src/binary/raw_binary_reader.rs

```diff
@@ -478,105 +504,78 @@ impl<R: IonDataSource> RawReader for RawBinaryReader<R> {
         let coefficient = self.read_int(coefficient_size_in_bytes)?;
 
         if coefficient.is_negative_zero() {
-            return Ok(Some(Decimal::negative_zero_with_exponent(exponent)));
-        }
-
-        Ok(Some(Decimal::new(coefficient.value(), exponent)))
-    }
-
-    fn read_big_decimal(&mut self) -> IonResult<Option<BigDecimal>> {
```

Comment:
PR tour: the read/write methods for the external `BigDecimal` type were deprecated in v0.6.1 in favor of the `Decimal` type.